groclient/client.py

```diff
@@ -1012,11 +1012,12 @@ def get_df(
 
         Parameters
         ----------
-            show_revisions : boolean, optional
-                False by default, meaning only the latest value for each period. If true, will return
-                all values for a given period, differentiated by the `reporting_date` field.
-            show_available_date : boolean, optional
-                False by default. If true, will return the available date of each data point.
+            reporting_history : boolean, optional
+                False by default, meaning only the latest value for each period. If true, will return all values for a given period 
+                which have a valid reporting_date.
```

Comment:
also I think we might be better keep `show_revisions` , but explaining it's deprecating and renamed to `reporting_history`

Reply:
added